docs/user/03-10-preconfigured-secret.md

```diff
@@ -43,12 +43,14 @@ The cluster ID represents a Kyma service instance created in a particular subacc
 
 ## Cluster Access
 
-By default, SAP BTP Operator has cluster-wide permissions. You cannot reconfigure the predefined settings.
+By default, SAP BTP Operator has cluster-wide permissions.
 
 The following parameters manage cluster access:
 
 | Parameter                     | Description                                                                                   |
 |-------------------------------|-----------------------------------------------------------------------------------------------|
 | **CLUSTER_ID**                | Generated when Kyma runtime is created.                                                       |
-| **MANAGEMENT_NAMESPACE**      | Always set to `kyma-system`.                                                |
+| **MANAGEMENT_NAMESPACE**      | By default, set to `kyma-system`.                                                |
 | **ALLOW_CLUSTER_ACCESS**      | You can use every namespace for your operations. The parameter is always set to `true`.<br>If you change it to `false`, the setting is automatically reverted. |
```

Comment:
@IwonaLanger please add here RELEASE_NAMESPACE too. Value: "By default, set to `kyma-system`."